merino-wikipedia/src/demo.rs

```diff
@@ -0,0 +1,24 @@
+use anyhow::{Context, Result};
+use merino_wikipedia::{ElasticHelper, WikipediaDocument, WikipediaNamespace};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let settings = merino_settings::Settings::load()
+        .await
+        .context("Loading settings")?;
+
+    let es_client = ElasticHelper::new(&settings.elasticsearch, "merino-scratch")?;
```

Comment:
Shall we also add the `index_name` to the settings?

Reply:
Probably just wanna defer this until this gets a formal `Provider`

Reply:
Yeah, I expect this will be a setting of the provider, in the same way that the Remote Settings collection is.